### ISSUE TITLE
Use AngleSharp HTML parser

### ISSUE THREAD

**Issue by axefrog** [SimpleBrowserDotNet/SimpleBrowser#88]
Hey guys, just came across this. Any benefit for SimpleBrowser? https://github.com/FlorianRappl/AngleSharp

**Comment by Teun** [comment]
Yeah, pretty cool. We currently expose the contents of the document as XElement. To make this non-breaking, we should at least output the tree again in XML form. But that should be possible.

**Comment by kevingy** [comment]
Is there something wrong with the parser we have?

**Comment by kevingy** [comment]
Hmmm... AngleSharp supports JavaScript.
